test/controllers/devise_token_auth/omniauth_callbacks_controller_test.rb

```diff
@@ -66,6 +66,11 @@ class OmniauthTest < ActionDispatch::IntegrationTest
       get_success
     end
 
+    test 'should be redirected via valid url' do
+      get_success
+      assert_equal 'http://www.example.com/auth/facebook/callback', request.original_url
```

Comment:
Without changes in this PR, this spec fails as `request.original_url` equals `'http://www.example.com//auth/facebook/callback'`.